locales/en.yml

```diff
@@ -20,8 +20,137 @@ posts:
+  styles: Dances
+  place: Your city
+  permission: Do you want free promo?
+  jobs: I can help with
+  contacts: Do you use social media?
+  contactFields:
+    instagram: instagram.com/
+    facebook: fb.com/
+    whatsapp: whatsapp.com/
+    telegram: t.me/
+    twitter: twitter.com/
+    tiktok: tiktok.com/
+    youtube: youtube.com/
+    email: Email
+    website: Website
+  genderList:
+    female: Female
+    male: Male
+    other: Other
+  daysList:
+    monday: Monday
+    tuesday: Tuesday
+    wednesday: Wednesday
+    thursday: Thursday
+    friday: Friday
+    saturday: Saturday
+    sunday: Sunday
+  meetingPlaces:
+    myPlace: My place
+    yourPlace: Your place
+    private: Somewhere private
+    public: Somewhere public
+  typeList:
+    dancer: Dancer
+    artist: Artist
+    organiser: Organiser
+    venue: Venue
+    city: City
+  objectivesList:
+    talk: Talk about dance
+    learn: Learn together
+    outdoors: Dance outdoors
+    indoors: Dance indoors
+    teach: Teach together
+    project: Dance project
+  profileSorts:
+    username: Username
+    registered: Registered
+    lastSeen: Last seen
+  placeholder:
+    username: username
+    whatsapp: whatsapp chat
+    business: for business requests
+    place: City
+    required: (Required)
+    story: Text (markdown)
+    dance: Style
+    objective: Objective
+    gender: Gender
+  description:
+    visibility: '- Public profiles are searchable in Google and used in our social media to attract new members.
+    - Members profile are only visible for logged-in users.
+    - Unlisted profiles are possible to open with exact link, but they are not listed in members lists and search.'
+    permission: 'Send us link to your profile on [Instagram](https://instagram.com/wedancevip) and we will promote it on our social media channels: Telegram, Instagram, Facebook and Twitter.'
+    learning: 'For example: Musicality in Salsa, Men Styling, etc.'
+    jobs: 'How can you help our community? What are you good at? For example: design, programming, marketing, social media, etc.'
+    birthday: We will show only your age'
+    weight: Leave this field blank if you think it's irrelevant.
+    partnerBio: 'What is important in your partner? Put it as a question, for example: Do you love dogs?'
+  tips:
+    styles: Mark up to 4 favorite dances that will be highlighted in your poster
+    story: 'Pitch yourself: Who are you? What do you offer? What do you want?
+
+
+    Tips for effective pitch:
+
+    - Uncomplicated: It should be catchy and roll off the tongue
+
+    - Concise: It shouldn’t take more than a minute to say or read
+
+    - Unique: It reflects your skills, goals, and desires
+
+    - Storyline: It covers who you are, what you offer, and where you want to be
+
+    - Appealing: Your elevator pitch is essentially a persuasive sales pitch; the emphasis should be on what you offer'
+  before:
+    username: Use only letters, numbers, underscores and periods.
```

Comment:
I would group `before`, `label`, `tips`, `after` by fieldName

Reply:
what about placeholder and descriptions?

Reply:
as well